### DISCUSSION TITLE
integrating with Supabase doesn't open the "link to vercel "popup.

### DISCUSSION THREAD

**Discussion by psyferblock** [vercel/community#3191] [Bug Report]
### Page to Investigate

https://vercel.com/integrations/supabase

### Steps to Reproduce

following the instructions of the site. I chose a Vercel account
pressed continue
specified the project  that is linked throughGgithub 
pressed continue 
it showed me the permissions required ( all clicked yes)
pressed add integration 
expected : a link that allows me to link my Supabase project with the Vercel project to be deployed 
actual link : https://app.supabase.com/sign-in?code=Yh54WEw7FA8FvoZ1esnKLKmf&configurationId=icfg_Bjgw3V26rI8NISGfD4UN0eFv&next=https%3A%2F%2Fvercel.com%2Fdashboard%2Fintegrations%2Ficfg_Bjgw3V26rI8NISGfD4UN0eFv%2Finstalled&source=marketplace

it redirects me to Supabase but doesn't show the page needed for integration
how can i deploy now ?

**Comment by angadveeersb** [comment]
i have this problem too

**Comment by amyegan** [comment]
I'm sorry to hear you're running into problems with the Supabase integration. It is not developed or maintained by Vercel, so I recommend reaching out to [Supabase Support](https://supabase.com/support) for help with it 🙇

**Comment by amyegan** [comment]
The integration is working again! 🎉 

https://github.com/vercel/community/discussions/3194#discussioncomment-6422129
